app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -129,31 +164,48 @@ export default {
         sku: null,
         price: '',
         currency: 'CLP',
-        quantity: '',
-        measure: '',
+        ingredient_measures_attributes: [],
       },
+      showingMeasureModal: false,
+      usedUnits: [],
     };
   },
-
+  methods: {
+    AddMeasure() {
+      const last_item = this.form.ingredient_measures_attributes[this.form.ingredient_measures_attributes.length -1];
+      if(last_item.name && last_item.quantity){
+        this.form.ingredient_measures_attributes.push({ name: undefined, quantity: undefined, id: undefined });
+      }
+    },
+    deleteUnit(unit) {
+      this.form.ingredient_measures_attributes = this.form.ingredient_measures_attributes
+      .filter((originalUnit) => originalUnit !== unit);
+    },
+  },
   created() {
     const {
       providerId,
       name,
       sku,
       price,
       currency,
-      quantity,
-      measure,
+      otherMeasures,
     } = this.ingredient;
-
+    var ingredient_measures_attributes; /* eslint-disable-line camelcase */
+    if(!otherMeasures){
+      ingredient_measures_attributes = [{ name: undefined, quantity: undefined, id: undefined }];
+    }else{
+      ingredient_measures_attributes = otherMeasures.data.map(function(unit) {
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Identifier 'ingredient_measures_attributes' is not in camel case  camelcase